content/Reducing_Power_Consumption.md

```diff
@@ -5,18 +5,19 @@ batteries.
 # Using the HALT Instruction
 
 The HALT instruction should be used whenever possible to reduce power
-consumption & extend the life of the batteries. This command stops the
+consumption and extend battery life. This instruction halts the
 system clock, reducing the power consumption of both the CPU and ROM.
 
-The CPU will remain stopped until an interrupt occurs at which point the
-interrupt is serviced and then the instruction immediately following the
-HALT is executed.
+The CPU will remain halted until an interrupt occurs, at which point the
```

Comment:
As noted by #174, this only applies to interrupts selected by `IE`; the actual wake-up condition is `IE & IF != 0`, regardless of `IME`. What `IME` changes, is whether the handler will be called.

(Therefore, in the line below, the "and [...] is enabled in IE" is actually misleading.)